code/modules/integrated_electronics/subtypes/logic.dm

```diff
@@ -212,6 +212,10 @@
 	if(comparable(A, B))
 		return A.data >= B.data
 
+/obj/item/integrated_circuit/logic/binary/greater_than_or_equal/old //for backwards compatibility
+	name = "greater_than or equal gate"
+	spawn_flags = 0
```

Comment:
I really don't like this "add another typepath for backwards compat" thing

Reply:
It lets old circuits work without affecting new ones, I don't think it's a big deal

Reply:
Many "not a big deal"s turn into an unmaintainable mess

Reply:
It would be really easy to add an out of game script to update old circuit pastes

Reply:
I don't really see how it makes things difficult to maintain

Reply:
Typepath bloat is a real thing, just do a remapping list in the decoder

Reply:
you're not actually fixing the issue if you knowingly put the issue as a subtype to coddlepeople who don't know find/replace.